index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <!-- <meta http-equiv="X-UA-Compatible" content="IE=edge" /> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav>
+      <div class="container">
+        <div class="name-info">Sunday Stanley</div>
+        <div class="menu-toggle">
+          <img src="./image/menu.png" alt="menu bar" />
+        </div>
+        <ul class="list-container">
+          <li>
+            <a href="">Portfolio</a>
+          </li>
+          <li>
+            <a href=""> About </a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+          <li>
+            <a href=""><img src="./image/msg.png" alt="msg icon" /></a>
+          </li>
+        </ul>
+      </div>
```

Comment:
- [ ] Kindly use `header` tag as the container for this section, and you can use `nav` tag inside it.